themes/theme-b2b/src/components/Link.styles.ts

```diff
@@ -6,6 +6,10 @@ export const Link: ThemeComponent<'Link'> = cva(
     variants: {
       variant: {
         menuItemLink: 'text-text-base hover:no-underline',
+        secondary: [
```

Comment:
There is a test in packages/components/src/Link/Link.test.tsx which could be adjusted. And I think there is a variant used which isn't actual any more?

Reply:
If you are referring to the "second" the theme is just a fixture for testing. We don't have any real themes in the tests.